docker-images/base-alpine-python36/harness.py

```diff
@@ -15,15 +15,16 @@
 from urllib3.util import parse_url
 
 
-def get_logger(name):
+def get_logger(name, id_name, id_val):
```

Comment:
Variable names id_name and id_val are a bit opaque.  How about corr_id_name, corr_id_val at a minimum.

Reply:
Agreed